src/github.com/getlantern/lantern-ui/app/js/controllers.js

```diff
@@ -186,16 +186,44 @@ app.controller('NewsfeedCtrl', ['$scope', '$rootScope', '$translate', function($
   $scope.hideNewsfeed = function(e) {
     $rootScope.showNews = false;
   };
-  $scope.hideNewsfeed();
+  $scope.showNewsfeed();
+
   $scope.feedUrl = function() {
     var mapTable = { 'fa': 'fa_IR' };
     var lang = $translate.use();
     lang = mapTable[lang] || lang;
     return "https://feeds.getiantem.org/" + lang + "/feed.json";
   };
+
+}]);
+
+app.controller('FeedTabCtrl', ['$scope', '$rootScope', '$translate', function($scope, $rootScope, $translate) {
+  $scope.tabActive = {};
+  $scope.selectTab = function (title) {
+    $scope.tabActive[title] = true;
+  };
+  $scope.deselectTab = function (title) {
+    $scope.tabActive[title] = false;
+  };
+  $scope.tabSelected = function (title) {
+    return $scope.tabActive[title] === true;
+  };
 }]);
 
 app.controller('FeedCtrl', ['$scope', 'gaMgr', function($scope, gaMgr) {
+  var copiedFeedEntries = [];
+  angular.copy($scope.feedEntries, copiedFeedEntries);
+  $scope.entries = [];
+  $scope.containerId = function($index) {
+    return "#feeds-container-" + $index;
+  };
+  $scope.addMoreItems = function() {
+    if ($scope.tabSelected($scope.feedsTitle)) {
```

Comment:
Add items only if tab is active, the key to make infinite scroll work for multiple tabs.